sdk/cognitivelanguage/azure-ai-language-conversations/tests/async/test_conv_summarization_async.py

```diff
@@ -14,10 +14,10 @@
 from azure.ai.language.conversations.aio import ConversationAnalysisClient
 
 
-class ConversationalIssueSummarizationAsyncTests(AsyncConversationTest):
+class ConversationalSummarizationAsyncTests(AsyncConversationTest):
 
     @GlobalConversationAccountPreparer()
-    async def test_conversational_issue_summarization(self, endpoint, key):
+    async def test_conversational_summarization(self, endpoint, key):
```

Comment:
I think the conv summarization tests are failing in CI because we changed their names here. We can rename the recordings or just re-record, up to you

Reply:
yeah i'll re-record after making all changes